sdk/resourcemanager/advisor/armadvisor/CHANGELOG.md

```diff
@@ -1,329 +1,9 @@
 # Release History
 
 ## 1.0.0 (2022-05-17)
-### Breaking Changes
 
-- Function `OperationEntityListResult.MarshalJSON` has been removed
-- Function `ConfigurationListResult.MarshalJSON` has been removed
-- Function `ResourceRecommendationBaseListResult.MarshalJSON` has been removed
-- Function `MetadataEntityListResult.MarshalJSON` has been removed
-- Function `SuppressionContractListResult.MarshalJSON` has been removed
-- Function `MetadataEntityProperties.MarshalJSON` has been removed
+The package of `github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/advisor/armadvisor` is using our next generation design principles since version 1.0.0, which contains breaking changes.
```

Comment:
next generation design principles link to https://azure.github.io/azure-sdk/general_introduction.html